src/index.html

```diff
@@ -2,16 +2,164 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="test-form"
+      autocomplete="off"
+    >
+      <fieldset class="fieldsetspace">
+        <legend> Personal information</legend>
+        <div class="fieldspace">
+          <label>
+            Surname:
+            <input type="text" name="surname">
+          </label>
+        </div>
+        <div class="fieldspace" >
```

Comment:
please add empty line between siblings
```suggestion
        </div>
        
        <div class="fieldspace" >
```